src/ripple/rpc/handlers/DepositAuthorized.cpp

```diff
@@ -77,6 +77,13 @@ Json::Value doDepositAuthorized (RPC::Context& context)
     if (!ledger)
         return result;
 
+    // If source account is not in the ledger it can't be authorized.
+    if (! ledger->exists (keylet::account(srcAcct)))
```

Comment:
Just curious, why do we use `! ledger-> exists(act)` here and `! ledger -> read(act)` on lines 88-89? Aren't they checking the same thing?